astropy/wcs/tests/test_wcs.py

```diff
@@ -386,15 +386,6 @@ def test_to_fits():
     assert header_string == wfits[0].header[-8:]
 
 
-def test_to_fits_1():
-    fits_name = get_pkg_data_filename('data/dist.fits')
-    w = wcs.WCS(fits_name)
-    wfits = w.to_fits()
-    assert isinstance(wfits, fits.HDUList)
-    assert isinstance(wfits[0], fits.PrimaryHDU)
-    assert isinstance(wfits[1], fits.ImageHDU)
-
-
```

Comment:
@astrofrog Was there a reason to remove this? I'm just wondering if it was an oversight. If so I'll add it back as I'm working on the tests now.